llvm/test/Transforms/SROA/strict-align-offset-transfer-2.ll

```diff
@@ -0,0 +1,128 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes
+; RUN: sed -e 's/@PTR_AS@/addrspace(200)/g' -e 's/@PTR_PAIR_SIZE@/32/g' %s \
```

Comment:
I meant use `addrspace(@PTR_AS@)` and `.p@PTR_AS@i8` so it's just `s/@PTR_AS@/200/` and `s/@PTR_AS@/0/`

Reply:
Ah yes that is slightly cleaner.